### DISCUSSION TITLE
Possible to use Sixel for full terminal experience?

### DISCUSSION THREAD

**Discussion by Skyro468** [pystardust/ani-cli#1002] [Ideas] [answered]
As title says, I am just throwing this idea out there as I recently came across Sixel, and have seen it being used to play youtube videos completely inside a terminal with no desktop environment. This is a video showing someone watching Nichijō's intro with a terminal. 
[Youtube video streaming with VT-340 compatible DEC SIXEL sequence (yaft on framebuffer, over SSH)](https://www.youtube.com/watch?v=hqMh47lYHlc)
Only downfall is only certain terminals support Sixel, so not sure if this idea is even worth looking into, but it does seem interesting to me at least. If I had enough experience with coding I would definitely be trying my hand at somehow getting this to work.
![image](https://user-images.githubusercontent.com/32147348/215310039-24182820-a651-4d35-8408-139e261d98c2.png)

**Comment by port19x** [comment, marked as answer]
This is a bad idea. Terminal image protocols have quite poor performance and I wouldn't want to troubleshoot the issues that could arise from this rather experimental usage.
I'd say make a PR, but it's likely not worth your time and would be an obscure env-var option at best
